arbeitszeit/use_cases/show_prd_account_details.py

```diff
@@ -19,10 +19,17 @@ class TransactionInfo:
     purpose: str
 
 
+@dataclass
+class PlotDetails:
+    dates: List[datetime]
```

Comment:
I think that the field name `dates` is missleading since the user would expect a `date` object. how about `timestamps`?